regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -27,8 +27,11 @@ var MainView = Backbone.View.extend({
 
     initialize: function() {
 
+
+        this.dataTable = null;
```

Comment:
Would it make sense for this to be an array, instead? I can imagine we'll want to have multiple tables in a single section.